src/app/components/About/About.jsx

```diff
@@ -71,21 +71,21 @@ All the materials in Digital Research Books Beta are completely free to read and
                     The material in Digital Research Books Beta are drawn from several
                                         public sources, mainly
               {' '}
-              <DS.UnderlineLink url="https://www.hathitrust.org/">HathiTrust</DS.UnderlineLink>
+              <DS.Link href="https://www.hathitrust.org/">HathiTrust</DS.Link>
```

Comment:
Is there a difference between the `href` and `url` attributes used on `DS.Link`. I see both and assume they do the same thing but maybe should be aligned for consistency?

Reply:
url is never used now, it's always href.

Reply:
Look down in the `License` file looks like its showing up there

Reply:
omg the license page wasn't working at all, thank you for catching this.